test/utils.test.js

```diff
@@ -35,48 +36,269 @@ describe('utils', () => {
         }
       ]
       expect(interleave(quasis, expressions)).toEqual(
-        '\n  display: block;\n  color: $color;\n  background: blue;\n'
+        '\n  display: block;\n  color: $dummyValue;\n  background: blue;\n'
       )
     })
+
+    it('converts interpolated expressions to dummy mixins', () => {
+      const quasis = [
+        {
+          value: {
+            raw: '\n  display: block;\n  '
+          }
+        },
+        {
+          value: {
+            raw: '\n  background: blue;\n'
+          }
+        }
+      ]
+      const expressions = [
+        {
+          name: undefined
+        }
+      ]
+      expect(interleave(quasis, expressions)).toEqual(
+        '\n  display: block;\n  -styled-mixin0: dummyValue;\n  background: blue;\n'
+      )
+    })
+
+    it('correctly converts several interpolations within a single property', () => {
+      const quasis = [
+        {
+          value: {
+            raw: '\n  display: block;\n  border: '
+          }
+        },
+        {
+          value: {
+            raw: ' '
+          }
+        },
+        {
+          value: {
+            raw: ' '
+          }
+        },
+        {
+          value: {
+            raw: ';\n  background: blue;\n'
+          }
+        }
+      ]
+      const expressions = [
+        {
+          name: 'borderWidth'
+        },
+        {
+          name: 'borderStyle'
+        },
+        {
+          name: 'color'
+        }
+      ]
+      expect(interleave(quasis, expressions)).toEqual(
+        '\n  display: block;\n  border: $dummyValue $dummyValue $dummyValue;\n  background: blue;\n'
+      )
+    })
+
+    it('correctly handles several interpolations in single line of css', () => {
+      const quasis1 = [
+        {
+          value: {
+            raw: '\n  display: '
+          }
+        },
+        {
+          value: {
+            raw: '; background: '
+          }
+        },
+        {
+          value: {
+            raw: ';\n'
+          }
+        }
+      ]
+      const expressions1 = [
+        {
+          name: 'display'
+        },
+        {
+          name: 'background'
+        }
+      ]
+      expect(interleave(quasis1, expressions1)).toEqual(
+        '\n  display: $dummyValue; background: $dummyValue;\n'
+      )
+
+      const quasis2 = [
+        {
+          value: {
+            raw: '\n  display: '
+          }
+        },
+        {
+          value: {
+            raw: '; '
+          }
+        },
+        {
+          value: {
+            raw: '\n'
+          }
+        }
+      ]
+      const expressions2 = [
+        {
+          name: 'display'
+        },
+        {
+          name: undefined
```

Comment:
Yeah so as I mentioned in the comments we don't actually need these anymore due to the `dummyValue`s but I think it's pretty clear so I left them.